### DISCUSSION TITLE
Add link support (or markdown) for stat_tables.

### DISCUSSION THREAD

**Discussion by Novail** [h2oai/wave#2006] [Ideas]
Hey, would it be possible to add markdowns/links to stat_tables? I get data from the web and would like to link the specific pages to the entries. 
(I can't imagine how much work it would be to change that, but wouldn't it be a good idea to allow labels, names, titles to use strings **and** links/markdowns at anytime?)

**Comment by mturoci** [comment]
Tracked in https://github.com/h2oai/wave/issues/2007. Thanks for bringing this up! :)
